From 00a13b953823757bb277f3fc9a6ac0a94b1726af Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Fri, 2 Oct 2020 17:53:20 +0200
Subject: [PATCH] Use u-boot 2016 custom fdt address

Let's increase the offset so the board
can boot using the custom fdt from rootfs.

Upstream-status: Inappropriate [configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 include/configs/p2771-0000.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/p2771-0000.h b/include/configs/p2771-0000.h
index 61310fb..ca995fa 100644
--- a/include/configs/p2771-0000.h
+++ b/include/configs/p2771-0000.h
@@ -41,7 +41,7 @@
 	"kernel_addr_r_size=02000000\0" \
 	"kernel_addr_r_aliases=loadaddr\0" \
 	"fdt_addr_r_align=00200000\0" \
-	"fdt_addr_r_offset=00000000\0" \
+	"fdt_addr_r_offset=06000000\0" \
 	"fdt_addr_r_size=00200000\0" \
 	"scriptaddr_align=00200000\0" \
 	"scriptaddr_offset=00000000\0" \
-- 
2.7.4

